documentation/sphinx/source/api-c.rst

```diff
@@ -718,6 +718,10 @@ Applications must provide error handling and an appropriate retry loop around th
 
    Most applications will not call this function.
 
+.. function:: FDBFuture* fdb_transaction_get_approximate_size(FDBTransaction* tr)
+
+   Retrieves the approximate transaction size so far in the returned future, which is the summation of the estimated size of mutations, read conflict ranges, and write conflict ranges. This can be called multiple times before transaction is committed.
```

Comment:
This block should mention how to extract the value from the future. I notice that we're currently relying on `fdb_future_get_version` for this, but my intuition says we shouldn't do that because the value we are getting isn't a version (even though the type is the same).

Reply:
Right. I'll add `fdb_future_get_int64` for this purpose.